openslides/motions/config_variables.py

```diff
@@ -0,0 +1,167 @@
+from django.core.validators import MinValueValidator
+from django.utils.translation import ugettext as _
+from django.utils.translation import pgettext, ugettext_lazy
+
+from openslides.core.config import ConfigVariable
+from openslides.poll.models import PERCENT_BASE_CHOICES
+
+from .models import Workflow
+
+
+def get_config_variables():
```

Comment:
Docstring: `Generator which yields all config variables of this app. They are grouped in 'General', 'Amendments', 'Supporters', 'Voting and ballot papers' and 'PDF'.  The generator has to be evaluated during app loading (see apps.py).`